src/csharp/Grpc.Core/Server.cs

```diff
@@ -89,29 +94,25 @@ public void AddServiceDefinition(ServerServiceDefinition serviceDefinition)
         /// Add a non-secure port on which server should listen.
         /// Only call this before Start().
         /// </summary>
-        public int AddListeningPort(string addr)
+        /// <returns>The port on which server will be listening.</returns>
+        /// <param name="host">the host</param>
+        /// <param name="port">the port. If zero, an unused port is chosen automatically.</param>
+        public int AddListeningPort(string host, int port)
```

Comment:
If they're different conceptually, I'd make the difference even *bigger* with different names, e.g. AddInsecureListener vs AddSecureListener. But I don't have enough context at the moment to know how much I care :)